vcd/config_test.go

```diff
@@ -119,6 +119,7 @@ type TestConfig struct {
 		NsxtManagerUsername string `json:"nsxtManagerUsername"`
 		NsxtManagerPassword string `json:"nsxtManagerPassword"`
 		NsxtManagerUrl      string `json:"nsxtManagerUrl"`
+		NsxtTier0Gateway    string `json:"nsxtTier0Gateway"`
```

Comment:
I think sample config update with this new field is missing

Reply:
Thanks, added!